sql/core/src/test/resources/sql-tests/inputs/explain-cbo.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TABLE t1(a INT, b INT) USING PARQUET;
+CREATE TABLE t2(c INT, d INT) USING PARQUET;
+
+ANALYZE TABLE t1 COMPUTE STATISTICS FOR ALL COLUMNS;
+ANALYZE TABLE t2 COMPUTE STATISTICS FOR ALL COLUMNS;
+
+SET spark.sql.cbo.enabled=true;
```

Comment:
We can add  `--SET spark.sql.cbo.enabled=true`  at the first line of this file.

Reply:
> We can add `--SET spark.sql.cbo.enabled=true` at the first line of this file.

Updated